packages/ilmomasiina-frontend/src/routes/Editor/components/Questions.tsx

```diff
@@ -155,13 +154,9 @@ const Questions = () => {
   });
 
   const updateOrder = useEvent(({ newIndex, oldIndex }: SortEnd) => fields.move(oldIndex, newIndex));
-
-  const keys = useShallowMemo(fields.value.map((item) => item.key));
-  const questionItems = useMemo(() => fields.map((name, i) => (
-    <QuestionRow key={keys[i]} name={name} index={i} remove={fields.remove} />
-  // This list only invalidates when the question positions or count change.
-  // eslint-disable-next-line react-hooks/exhaustive-deps
-  )), [keys]);
+  const questionItems = fields.map((name, i) => (
+    <QuestionRow key={fields.value[i].key} name={name} index={i} remove={fields.remove} />
+  ));
```

Comment:
This existed to prevent the sortable HOC from re-rendering every time, which was a major perf bottleneck on the editor. Now it'll recreate the children and thus rerender from scratch every time